gtk/src/auto/header_bar_accessible.rs

```diff
@@ -0,0 +1,196 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir)
+// from gir-files (https://github.com/gtk-rs/gir-files)
+// DO NOT EDIT
+
+use glib::object::Cast;
+use glib::object::IsA;
+use glib::StaticType;
+use glib::ToValue;
+use std::fmt;
+
+glib::wrapper! {
+    #[doc(alias = "GtkHeaderBarAccessible")]
+    pub struct HeaderBarAccessible(Object<ffi::GtkHeaderBarAccessible, ffi::GtkHeaderBarAccessibleClass>) @extends atk::Object;
+
+    match fn {
+        type_ => || ffi::gtk_header_bar_accessible_get_type(),
+    }
+}
+
+impl HeaderBarAccessible {
+    pub const NONE: Option<&'static HeaderBarAccessible> = None;
+
+    // rustdoc-stripper-ignore-next
+    /// Creates a new builder-pattern struct instance to construct [`HeaderBarAccessible`] objects.
+    ///
+    /// This method returns an instance of [`HeaderBarAccessibleBuilder`](crate::builders::HeaderBarAccessibleBuilder) which can be used to create [`HeaderBarAccessible`] objects.
+    pub fn builder() -> HeaderBarAccessibleBuilder {
+        HeaderBarAccessibleBuilder::default()
+    }
+}
+
+#[derive(Clone, Default)]
+// rustdoc-stripper-ignore-next
+/// A [builder-pattern] type to construct [`HeaderBarAccessible`] objects.
+///
+/// [builder-pattern]: https://doc.rust-lang.org/1.0.0/style/ownership/builders.html
+#[must_use = "The builder must be built to be used"]
+pub struct HeaderBarAccessibleBuilder {
+    accessible_description: Option<String>,
+    accessible_name: Option<String>,
+    accessible_parent: Option<atk::Object>,
+    //accessible-role: /*Unknown type*/,
```

Comment:
Could probably be generated as well if `atk::RelationType` is added to the manual list

Reply:
Yeah but not as part of the backport, that should happen first on master :)

Reply:
See https://github.com/gtk-rs/gtk3-rs/pull/731